soc/xtensa/intel_adsp/ace/multiprocessing.c

```diff
@@ -170,6 +181,7 @@ int soc_adsp_halt_cpu(int id)
 		return -EINVAL;
 	}
 
+	ACE_PWRCTL->wpdsphpxpg &= ~BIT(id);
```

Comment:
What is wpdsphpxpg?

Reply:
Commit message is describing what this bitfield does. In the future, it should be done with usage of power_domain_intel_adsp.

Reply:
When you read a file and try to understand what it does, you don't do git blame to read all the commit messages. The power control code is full of names that look like the result of obfuscation (eg. wpdsphpxpg, wphstpg, wphubhppg, wpdspulppg, wpioxpg, wpmlpg, phubulppg, dsphpxpgs, hstpgs, hubhppgs, dspulppgs, ioxpgs, mlpgs, hubulppgs). We should definitely add comments with human names, add one-sentence descriptions of what we're doing when toggling some bit. Without it, it's very easy to make a mistake and code maintenance becomes impossible.